Control/public/common/detectorModal.js

```diff
@@ -32,20 +32,24 @@ const detectorsModal = (model) =>
         model.detectors.listRemote.match({
           NotAsked: () => null,
           Loading: () => h('.w-100.text-center', loading(2)),
-          Success: (data) => detectorsList(model, data),
+          Success: (data) => detectorsList(model, model.session.admin ?
+            data : data.filter((det) => model.detectors.roles.includes(det))),
           Failure: (_) => h('.w-100.text-center.danger', [
-            iconCircleX(), ' Unable to load list of detectors. Use GLOBAL View'
+            iconCircleX(), ' Unable to load list of detectors.'
           ])
         })
       ]),
       h('.w-100.pv3.f3.flex-row', {style: 'justify-content:center;'},
         h('.w-50.flex-column.dropdown#flp_selection_info_icon', [
-          h(`button.btn.btn-default.w-100`, {
+          (model.session.admin || model.detectors.roles.includes('GLOBAL'))
```

Comment:
Is this supposed to be similar to a ternary operator?
If condition is met we go with `GLOBAL`, otherwise with `GUEST`?
If yes, can we use the ternary operator?

Or even better, we can define an individual node for the button and we send as parameter `GUEST` or `GLOBAL` as they are pretty much the same